cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -72,10 +75,21 @@ import qualified Ouroboros.Network.Protocol.LocalStateQuery.Type as LocalStateQu
 import qualified Shelley.Spec.Ledger.API.Protocol as Ledger
 import qualified System.IO as IO
 
+import qualified Data.ByteString.Base16 as B16
+
+import           Ouroboros.Consensus.Block (ConvertRawHash (..))
+import           Ouroboros.Network.Block (HeaderHash)
+import           Ouroboros.Consensus.Cardano.Block (CardanoBlock)
+
 {- HLINT ignore "Reduce duplication" -}
 {- HLINT ignore "Use const" -}
 {- HLINT ignore "Use let" -}
 
+-- | Hex encode and render a 'HeaderHash' as text.
+renderHeaderHash :: (ConvertRawHash blk, blk ~ CardanoBlock StandardCrypto) => proxy blk -> HeaderHash blk -> Text
```

Comment:
I think it makes more sense to wrap `HeaderHash blk` in `cardano-api` and implement a `SerialiseAsRawBytes` instance instead of exposing `ouroboros-network` types in the cli.

Reply:
I'm no longer using this ype and I don' need the function anymore.